### PR TITLE
Remove virus spread through blood and gibs

### PR DESCRIPTION
**Pull request by FlameArrow57**
[MEDICAL][REMOVAL][PERFORMANCE]

## About the PR 
This PR removes a system in the game where airborne viruses can spread through blood decals

Also removes unused Edison's Disease.

Testing on Cogmap 2 with about 50 people in addition to roundstart mobs for two to three minutes:

Without PR:
image

With PR:
image

## Why's this needed? 
At the moment, the only case where a virus can be added to blood decals is through a mob being gibbed. Considering that has been in the game a while, the number of realistic cases where a virus should be able to spread, and the current limited interactions with viruses that are currently in the game, I feel it would make more sense to just remove the feature right now as any more work on something like this would need an improved rework if anything more significant is to be done on it.

### CODE REVIEW

File: code/mob.dm [hunk 1]
```diff
@@ -1888,17 +1883,15 @@
...
 
 	if (animation)
 		animation.delaydispose()
 	qdel(src)
 
 /mob/proc/partygib(give_medal)
 	if (isobserver(src))
-		var/list/virus = src.ailments
-		partygibs(src.loc, virus)
+		partygibs(src.loc)
```
Comment (frawhst) [inline code/mob.dm:1894]: who's out here giving ghosts diseases

File: code/modules/tutorials/blob_tutorial.dm [hunk 2]
```diff
@@ -251,7 +251,7 @@
 				sleep(4 SECONDS)
 				W.sprayAt(locate(tx, ty + 5, tz), 8)
 				sleep(4 SECONDS)
-				gibs(get_turf(W), list(), list())
+				gibs(get_turf(W), list())
```
Comment (frawhst) [inline code/modules/tutorials/blob_tutorial.dm:254]:
```suggestion
				gibs(get_turf(W))
```
`list()` isn't necessary, `null` is handled fine

File: code/modules/tutorials/blob_tutorial.dm [hunk 3]
```diff
@@ -328,7 +328,7 @@
...
```
Comment (frawhst) [inline code/modules/tutorials/blob_tutorial.dm:331]:
```suggestion
				gibs(get_turf(W))
```
.

File: code/obj/decal/cleanable.dm [hunk 4]
```diff
@@ -7,9 +7,9 @@
 ////////////////
 
 ////////////////
-proc/make_cleanable(var/type,var/loc,var/list/viral_list)
+proc/make_cleanable(var/type,var/loc)
 	RETURN_TYPE(/obj/decal/cleanable)
-	return new type(loc, viral_list)
+	return new type(loc)
```
Comment (frawhst) [inline code/obj/decal/cleanable.dm:12]: Isn't this proc just a `new` wrapper now (ie completely unnecessary). Out of scope for this PR but should be noted!!!

**Review by frawhst — APPROVED** [review]
fine by me

File: code/mob.dm (hunk 1 above)
Comment (FlameArrow57) [reply code/mob.dm:1894]: :p

**Pull request merged**